lib/loader/index.js

```diff
@@ -135,6 +186,16 @@ module.exports = {
       var firstScript = getFirstScript();
 
       firstScript.parentNode.insertBefore(script, firstScript);
+
+      _loadedUrls[url] = true;
+
+      if (options.namespaceName) {
+        namespace.loadedUrls = _loadedUrls;
```

Comment:
Not quite. The first time you set that property on the namespace, you have to register it. Not afterwards.

So it might be better for you to, at the outset and once, register an object on the namespace, then add loadedUrls to that object as you need to elsewhere in the code.